tests/baselines/reference/duplicateObjectLiteralProperty_computedName.errors.txt

```diff
@@ -0,0 +1,27 @@
+tests/cases/compiler/duplicateObjectLiteralProperty_computedName.ts(3,5): error TS2300: Duplicate identifier '[1]'.
+tests/cases/compiler/duplicateObjectLiteralProperty_computedName.ts(8,5): error TS2300: Duplicate identifier '[-1]'.
+tests/cases/compiler/duplicateObjectLiteralProperty_computedName.ts(13,5): error TS2300: Duplicate identifier '[+1]'.
+
+
+==== tests/cases/compiler/duplicateObjectLiteralProperty_computedName.ts (3 errors) ====
+    const a = {
+        1: 0,
+        [1]: 0 // Duplicate
+        ~~~
+!!! error TS2300: Duplicate identifier '[1]'.
+    }
+    
+    const b = {
+        "-1": 0,
+        [-1]: 0 // Duplicate
+        ~~~~
+!!! error TS2300: Duplicate identifier '[-1]'.
+    }
+    
+    const c = {

```

Comment:
I don't think this should be an error; at least on node it's fine:

```
> { "+1": 1, [+1]: 2 }
{ '1': 2, '+1': 1 }
```

Reply:
Yep, this case should instead be `"1"/[+1]` and also `1/[+1]`.

Reply:
@sandersn Thanks for the feedback